### ISSUE TITLE
Add a last updated info as footer

### ISSUE THREAD

**Issue by utkarsh867** [rcltech/sls#34]
As a user, I want to be able to know when the website data was last updated so that I can know that the information is latest.

**Comment by welvin21** [comment]
something like the one in your personal website?? and should it be an automated update? (i.e. when travis deploys, change the date in the footer)

**Comment by utkarsh867** [comment]
> something like the one in your personal website?? and should it be an automated update? (i.e. when travis deploys, change the date in the footer)

Yes! The updated data should come from backend, when washer data comes in.

**Comment by 98sean98** [comment]
This can implemented relatively quickly with Phoenix.

Each Washer now has an updatedAt field.
